app/styles/design-system/interactions/button.css

```
@@ -1,16 +1,16 @@
 button[data-appearance="ghost"]:not(:disabled, [aria-disabled="true"]) {
     &:hover {
-        --background-color: color-mix(in srgb, currentColor 15%, var(--foreground-on-surface));
+        --background-color: color-mix(in srgb, currentcolor 15%, var(--foreground-on-surface));
     }
 
     &:focus-visible {
-        --background-color: color-mix(in srgb, currentColor 40%, var(--foreground-on-surface));
+        --background-color: color-mix(in srgb, currentcolor 40%, var(--foreground-on-surface));
         outline-offset: var(--spacing-sm);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)